ClosedXML_Tests/Excel/CalcEngine/MathTrigTests.cs

```diff
@@ -450,24 +450,24 @@ public void Base_ThrowsValueExceptionOnAnyInputNotANumber(string input, string t
                     minLength)));
         }
 
-        [TestCase(24.3, 5, 25)]
-        [TestCase(6.7, 1, 7)]
-        [TestCase(-8.1, 2, -8)]
-        [TestCase(5.5, 2.1, 6.3)]
-        [TestCase(5.5, 0, 0)]
-        [TestCase(-5.5, 2.1, -4.2)]
-        [TestCase(-5.5, -2.1, -6.3)]
-        [TestCase(-5.5, 0, 0)]
-        public void Ceiling(double input, double significance, double expectedResult)
-        {
-            var actual = (double)XLWorkbook.EvaluateExpr($"CEILING({input}, {significance})");
```

Comment:
Isn't it better just to leave the tests as is, but replace this line with e.g.
```c#
var actual = (double)XLWorkbook.EvaluateExpr($"CEILING({input.ToInvariantString()}, {significance.ToInvariantString()})");
```

Reply:
I guess you're right